### PR TITLE
feat: Add support for performance_insights_kms_key_id

### LINKED ISSUE #238: Support performance_insights_kms_key_id
https://www.terraform.io/docs/providers/aws/r/db_instance.html#performance_insights_kms_key_id

### PR DESCRIPTION
**Pull request by zahorniak**
## Description
Added missing parameter [performance_insights_kms_key_id](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#performance_insights_kms_key_id) for resource `aws_db_instance`

## Motivation and Context
This pull request closes #238 

## Breaking Changes
This change doesn't break backwards compatibility with the current release

## How Has This Been Tested?
I've tested it by running `complete-postgres` example.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -357,6 +357,12 @@ variable "performance_insights_retention_period" {
   default     = 7
 }
 
+variable "performance_insights_kms_key_id" {
+  description = "The ARN for the KMS key to encrypt Performance Insights data."
+  type        = string
+  default     = ""
```
Comment (bryantbiggs) [inline variables.tf:363]: could we set the default as `null` please

File: modules/db_instance/variables.tf [hunk 2]
```diff
@@ -280,6 +280,12 @@ variable "performance_insights_retention_period" {
   default     = 7
 }
 
+variable "performance_insights_kms_key_id" {
+  description = "The ARN for the KMS key to encrypt Performance Insights data."
+  type        = string
+  default     = ""
```
Comment (bryantbiggs) [inline modules/db_instance/variables.tf:286]: likewise, `null` as the default please

File: main.tf [hunk 3]
```diff
@@ -100,6 +100,7 @@ module "db_instance" {
 
   performance_insights_enabled          = var.performance_insights_enabled
   performance_insights_retention_period = var.performance_insights_retention_period
+  performance_insights_kms_key_id       = var.performance_insights_kms_key_id
```
Comment (bryantbiggs) [inline main.tf:103]:
```suggestion
  performance_insights_kms_key_id       = var.performance_insights_enabled == true ? var.performance_insights_kms_key_id : null
```

**Comment by zahorniak** [comment]
Hi @bryantbiggs,
I updated variables as you requested.

**Review by bryantbiggs — APPROVED** [review]
verified - thanks for the PR @zahorniak - @antonbabenko 👍🏼

**Comment by antonbabenko** [comment]
Thanks @zahorniak !

[v2.22.0](https://github.com/terraform-aws-modules/terraform-aws-rds/releases/v2.22.0) has been just released.

**Pull request merged**